static/templates/settings/data-exports-admin.handlebars

```diff
@@ -0,0 +1,19 @@
+<div id="data-exports" class="settings-section" data-name="data-exports-admin">
+    {{#if is_admin}}
+    <div class="alert" id="export_status" role="alert">
+        <span class="export_status_text"></span>
+    </div>
+    <form class="form-horizontal">
+        <p class="alert-word-settings-note">{{t "Upload the organization's public-only data directly to  an S3 backend.
+        Or, if no S3 configuration exists, the data will be stored on disk in the specified directory on the Zulip server."}}</p>
```

Comment:
Change this description to not talk about the S3/LOCAL_UPLOADS_DIR issue -- we're going to make it work regardless.

I think actually this whole alert can be replaced with "This tool does a <a href="/help/export-your-organization">public export</a> of your organization's history."